utilities/DumpUnitaryTest/UnitTest1.cs

```diff
@@ -0,0 +1,21 @@
+using System;
```

Comment:
Could you please add the copyright header to this file, per the guidance at https://docs.opensource.microsoft.com/content/releasing/copyright-headers.html#c-2 ?